### ISSUE TITLE
Introduce Private Accounts

### ISSUE THREAD

**Issue by imRohan** [imRohan/Pantry#44]
Allow users to create private accounts, which require some form of auth
(Bearer Token Auth or other) in order to access their contents/basket contents.

**Comment by IyaadDev** [comment]
Yes, that's true, a Pantry ID isn't really like a private token, and anyone with the ID has basically a master API Key, a Bearer Access Token would be smart for more confidential data
